src/Migrations/Version20220322144548.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\DataHubBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use Pimcore\Bundle\DataHubBundle\Configuration;
+
+final class Version20211108160248 extends AbstractMigration
```

Comment:
- migration actually needs to be located in  Pimcore\Bundle\DataHubBundle\Migrations\PimcoreX namespace
- filename and classname need to match
- should extend from `BundleAwareMigration`  ... to make sure it is not executed when bundle is not installed

Reply:
> * migration actually needs to be located in  Pimcore\Bundle\DataHubBundle\Migrations\PimcoreX namespace
> * filename and classname need to match
> * should extend from `BundleAwareMigration`  ... to make sure it is not executed when bundle is not installed

As discussed, I changed the checkbox from "Enable Introspection" to "Disable Introspection". Default: Introspection is enabled.